### ISSUE TITLE
Navigation Consistency

### ISSUE THREAD

**Issue by inderpreetsingh** [BRL-CAD/web#7]
There are various inconsistency issues, but the major one is navigation.

**Comment by brlcad** [comment]
Agreed.  My intention was to wire up the main menu to existing resources as a first step, so it can go live immediately, and then get them working by first jumping down the page for a better workflow.  Your thoughts?

**Comment by inderpreetsingh** [comment]
I have shared some stuff on IRC, so sharing the same..

> I found this on my server. http://ishwerdas.com/brlcad/ It has almost all the issues solved (except moving signup page below). 

And another minor ones, but mostly quick fixes. 

> I think I created it much before GSoC, or before I designed the wiki theme (http://beta.brlcad.org/wiki/Main_Page) as a prototype. It's consistent, it looks good and it just works. Why don't we use this instead?
> 
> I understand the part about using GCI stuff, but for this HTML based temporary landing page, I feel like it would be a duplication of effort. We would still be using GCI created Wordpress theme anyway, so GCI stuff is still being used. I can do some changes in this (would take an hour max) and we are good to go, what say?
>

**Comment by brlcad** [comment]
I suggest merging your changes into the /wp files as a starting point since your version has several fixes still needed.

As for the main page, I think it'll still be more work to fix the WP page than to get this custom landing page working and linked up.  Provided details on IRC, but it's something we can sort through later.

We do still want to use WP for news articles, so I suggest the focus be there.  The landing page has a NEWS link -- that should jump down the page to recent articles, but then provide a button there that goes to /wp for all articles.  I was going to simply point it at the current brlcad.org/d Drupal install, but getting WP working and loaded with the current articles would be much better.

**Comment by peter-sa** [comment]
Since we're talking about the landing page choices:

Looking at the index.html right now, the way that news articles are written/displayed in it looks like it might not be ideal, since it makes dynamically updating them quite hard (and the current implementation uses a lot of markup and javascript duplicated for each article...).  Do we need to eventually combine the index page with wordpress in order to get the current articles? We might also be able to use the WP API to dynamically pull them into the front page.

A few things stood out to me when I was playing around with the current setup, here are a few design ideas:

From a usability standpoint, I also think that it might be a good idea to put at the very least a title as well as a thumbnail image next to each news article, I found the current appearance quite confusing.

Since the front page is quite short, do we even want the nav links to jump to a page section that then has more links for going to the actual related pages?  If we can get the wordpress, wiki, and front page menus to all line up, it might be worth it to just have "Documentation" go straight to the wiki on documentation, "News" go straight to the blog, etc.  Then the content on the index page is a brief summary of the recent/important parts of each of those categories, with another set of links to the real pages.

**Comment by brlcad** [comment]
Yes, we'll want to dynamically pull them in later and at least some of the menu items should just jump to the main content (e.g., Documentation).  I hadn't really gotten to anything below the three feature highlights under the title, but agree that the simple circles are a bit too simplistic.  I have some examples in mind, but was going to worry about that after this goes live since it's not a show stopper.

**Comment by inderpreetsingh** [comment]
> As for the main page, I think it'll still be more work to fix the WP page than to get this custom landing page working and linked up. Provided details on IRC, but it's something we can sort through later.

The link I showed http://ishwerdas.com/brlcad/ is not a wordpress page. It's a simple HTML Page. I was suggesting to use it in place of current landing page so that we can quickly get it online.

**Comment by brlcad** [comment]
I provided a list of things on IRC that would need to change... I can't see it taking less work to clean it up at this point.  The logo, the login, and non-brl-cad images would need to change at a minimum. That said, your page definitely has a better highlights section and persistent hovering menu that we should incorporate.

**Comment by inderpreetsingh** [comment]
Okay, got it.

**Comment by inderpreetsingh** [comment]
I still suspect :P the current one taking more time so I would keep the other ready too in case we change our mind.
